src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,25 @@
+/* eslint-disable implicit-arrow-linebreak */
+
 import React from 'react';
+import commentsFromServer from '../../api/comments';
+import { Comment } from '../../types/Comment';
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
+
+type Props = {
+  postId: number,
+};
+
+export const CommentList: React.FC<Props> = ({ postId }) => {
+  const found = (element: Comment) => element.postId === postId;
+  const filtered = commentsFromServer.filter(found);
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
-);
+  return (
+    <div className="CommentList">
+      {filtered.length > 0
+        ? filtered.map(el =>
+          <CommentInfo name={el.name} email={el.email} body={el.body} />)
```

Comment:
Add a unique key prop to the `CommentInfo` component, for example: `<CommentInfo key={el.id} name={el.name} email={el.email} body={el.body} />`.